apis/python/tests/test_dense_nd_array.py

```diff
@@ -114,6 +114,24 @@ def test_dense_nd_array_reshape(tmp_path):
         assert a.reshape((100, 10, 1))
 
 
+@pytest.mark.parametrize("appendable", [True, False])
```

Comment:
the test is fine, but the naming/concept of "appendable" is misleading.

The test should simply confirm that DenseNDArray requires a shape to be specified.  I.e., there is no defaulting on the shape param to create().